docker-compose.yml

```diff
@@ -0,0 +1,48 @@
+version: "3.5"
+services:
+  db:
+    image: postgres
+    environment:
+      POSTGRES_USER: throat
+      POSTGRES_PASSWORD: dev
+      POSTGRES_DATABASE: throat
```

Comment:
```suggestion
      POSTGRES_DATABASE: throat
    healthcheck:
      test: ["CMD-SHELL", "pg_isready -U postgres"]
      interval: 5s
      timeout: 5s
      retries: 5
```

Reply:
Got it, this looks reasonable. I will get that incorporated.

Reply:
Arg. This made it fail for me locally where it was working before :(

Reply:
https://stackoverflow.com/questions/31746182/docker-compose-wait-for-container-x-before-starting-y/41854997#41854997

Reply:
https://docs.docker.com/compose/startup-order/

Reply:
I added an additional command to the migration service for `restart:on_failure` this is working locally for me and I also kept the healthcheck in.